src/arrayMethodJoin.js

```diff
@@ -5,7 +5,42 @@
  */
 function applyCustomJoin() {
   [].__proto__.join2 = function(separator) {
```

Comment:
you can add default value to separator
```suggestion
  [].__proto__.join2 = function(separator = ',') {
```